--- a/src/base/stats/SConsopts
+++ b/src/base/stats/SConsopts
@@ -41,11 +41,12 @@
     # include path and library path provided by pkg-config. We perform
     # this check even if there isn't a pkg-config configuration for hdf5
     # since some installations don't use pkg-config.
-    conf.env['CONF']['HAVE_HDF5'] = \
-            conf.CheckLibWithHeader('hdf5', 'hdf5.h', 'C',
-                                    call='H5Fcreate("", 0, 0, 0);') and \
-            conf.CheckLibWithHeader('hdf5_cpp', 'H5Cpp.h', 'C++',
-                                    call='H5::H5File("", 0);')
+    conf.env['CONF']['HAVE_HDF5'] = 0
+#\
+#            conf.CheckLibWithHeader('hdf5', 'hdf5.h', 'C',
+#                                    call='H5Fcreate("", 0, 0, 0);') and \
+#            conf.CheckLibWithHeader('hdf5_cpp', 'H5Cpp.h', 'C++',
+#                                    call='H5::H5File("", 0);')

     if conf.env['CONF']['HAVE_HDF5']:
         conf.env.TagImplies('hdf5', 'gem5 lib')
